### ISSUE TITLE
Implement support for GA4

### ISSUE THREAD

**Issue by philc** [denoland/ga#8]
Universal Analytics will [stop processing data](https://support.google.com/analytics/answer/11583528) on July 1, 2023.

**Comment by scaledrop** [comment]
Yes, We're running out of time. @kitsonk @piscisaureus

**Comment by ghost** [comment]
Yep we have run out of time for now. GA4 Server side tagging seems to complicated to implement. I think something like a custom analytics service seems like a better option. Similar to cloudflare server side analytics.
